static/include/app/apis/overrides/protos/data.ExportTabularData.md

```diff
@@ -0,0 +1 @@
+Obtain unified tabular data and metadata from the specified data source.
```

Comment:
For my own understanding, does this need to be here because the automation wouldn't pull this description from the Python SDK and only looks to the proto which doesn't have this description? Wouldn't it be better to edit https://github.com/viamrobotics/api/blob/3f0b40a80d223721fce7e385afe026aeeb6bd4fc/proto/viam/app/data/v1/data.proto#L24 instead of creating an override?

Reply:
yes the automation requires it

Reply:
and yes this is not ideal but not something that's easy to fix at the moment.